apps/src/tutorialExplorer/filterSet.jsx

```diff
@@ -10,12 +10,21 @@ const FilterSet = React.createClass({
     filterGroups: React.PropTypes.array.isRequired,
     onUserInput: React.PropTypes.func.isRequired,
     selection: React.PropTypes.objectOf(React.PropTypes.arrayOf(React.PropTypes.string)).isRequired,
+    mobileLayout: React.PropTypes.bool.isRequired,
     roboticsButtonUrl: React.PropTypes.string
   },
 
+  componentDidMount() {
+    if (this.props.mobileLayout) {
+      $('html, body').animate({
+        scrollTop: $("#filterset").offset().top - $("#filterheader").height() - 6
+      }, 1000);
```

Comment:
We'd prefer not to jquery to access our DOM elements if possible. In the case of filter set, we can do this using refs instead.

i.e. on line 27, we do
```jsx
<div ref={filterSet => this.filterSet = filterSet}>
```
and then here we can do
```js
scrollTop: $(this.filterSet).offset().top - $("#filterheader").height() -6
```

It's less immediately clear to me what to do about filterheader.

Also, it would be good to add a comment explaining the magic number 6 that we're subtracting.